reports/technical_report/5_conclusions.md

```diff
@@ -0,0 +1,62 @@
+
+<!---
+Alex add here about the opportunities of limitations of other sources
+-->
+
+## Implications
+
+Our analysis of exposure and diversification to Covid-19 in Scotland suggests that the industrial composition of the country makes it slightly more succeptible to the economic impacts of Covid-19 than other parts of the UK: sectors related to tourism such as *Accommodation* and *Travel agencies* and cultural activities such as *Libraries and Museums* where Scotland has a relative specialisation are highly exposed to Covid-19 and have limited opportunities for diversification from it. 
+
+This is reflected in important geograaphical heterogeneity in exposure to Covid-19. While council areas in the Highlands and Islands appear to be more exposed to Covid-19 because of the relative importance for their economies of industries such as those we described above, we also see council areas in other parts of Scotland showing high levels of exposure to the pandemic as a consequence of their industrial particularities. Two examples of this are City of Edinburgh at the onset of the pandemic because of its reliance on finance and professional services, and Aberdeen City and Aberdeenshire because of the importance of oil and energy industries that have, in some cases, limited opportunities for diversification into sectors which are less exposed to Covid-19. Local authorities in the South and Southwest which more manufacturing activity appear, in general, less exposed to the pandemic.
+
+Our analysis of the evolution of normalised claimant counts at the council area level and their correlation with other secondary data suggests that wealthier local economies with better educated workforces have suffered a stronger shock from Covid-19. At the same time, these areas are likely to have access to the resources and capabilities to overcome this shock, not least the skills to diversify into industries that are less exposed to the pandemic. Many of the digitalised and knowledge intensive sectors that our analysis suggests are less exposed and more able to move into new markets are more important in those areas. An important challenge for policymakers will be to put in place interventions to support industries with less diversification opportunities whitstand the shocks of the pandemic while, at the same time, boosting their ability to transition into new industries while mitigating potential increases in inequality (many of the industries that are highly exposed to the pandemic and have low diversification opportunities are low skill, low salary and low productivity). This will be important for boosting Scottish' local economies resilience to Covid-19 in the short term, and to other future shocks linked to Brexit, automation and the environmental transition that may be coming down the line. The analysis that we have undertaken here underscores the potential of novel, high granularity data sources for informing such policies.
```

Comment:
It is most likely the poorer people in the wealthy economies who suffer the shock.